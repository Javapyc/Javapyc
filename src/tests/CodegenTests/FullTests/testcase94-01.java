class Main
{
   public static void main( String[] args )
   {
      int a = 1;
      int b = 20;
      int c = -5;
      int d = -200;
      
      a = b * c - a;
      
      b = a/d;
      
      c = b + a * c;
      
      System.out.println(a);
      
      if(b>0)
      {
    	  System.out.println(a + b);
      }
      else
      {
    	  System.out.println(c/2);
      }
      
      System.out.println(c);
      
     while(c > 500)
     {
    	 System.out.println(c * 4);
    	 c = c - 1;
     }
      
   }
}
