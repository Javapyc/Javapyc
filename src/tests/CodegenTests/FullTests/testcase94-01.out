-101
252
505
2020
2016
2012
2008
2004
